common/src/test/scala/uk/ac/wellcome/platform/archive/common/fixtures/BagBuilder.scala

```diff
@@ -47,7 +47,8 @@ trait BagBuilder[BagLocation <: Location, BagPrefix <: Prefix[BagLocation], Name
     space: StorageSpace = createStorageSpace,
     externalIdentifier: ExternalIdentifier = createExternalIdentifier,
     version: BagVersion = BagVersion(randomInt(from = 2, to = 10)),
-    payloadFileCount: Int = randomInt(from = 5, to = 50)
+    payloadFileCount: Int = randomInt(from = 5, to = 50),
+    bucketName: String = randomAlphanumeric
```

Comment:
```suggestion
    bucketName: String = createBucket
```

Not all random alphanumeric strings are valid bucket names, and you can get flaky test failures depending on whether or not the random string is valid.

Reply:
thanks